src/openapi_python_generator/language_converters/python/templates/aiohttp.jinja2

```diff
@@ -24,12 +24,12 @@ async def {{ operation_id }}({{ params }}) -> {% if return_type.type is none or
                 raise HTTPException(inital_response.status, f'{{ operationId }} failed with status code: {inital_response.status}')
             response = await inital_response.json()
 
-{% if return_type.complex_type %}
-{% if return_type.list_type is none %}
+{% if return_type.type is none or return_type.type.converted_type is none %}
+            return response.json()
```

Comment:
I think `return None` is the correct approach here, instead of the dictionary, as the return type is also typed as None in this case afaik. It should be also in line to what the Openapi definition actually provides.

Reply:
My bad, that is actually what I originally intended. I got confused by the logic inversion. Fixed the commit.